static_src/actions/org_actions.js

```diff
@@ -26,7 +27,14 @@ const orgActions = {
       orgGuid
     });
 
-    return cfApi.fetchOrg(orgGuid).then(orgActions.receivedOrg);
+    return cfApi.fetchOrg(orgGuid)
+      .then(orgActions.receivedOrg)
+      .catch((err) =>
+        errorActions.importantDataFetchError(
+          err,
+          'organization data may be incomplete'
+        )
```

Comment:
So what would the benefit be of that? It looks like that doesn't require the `err` being passed?